### DISCUSSION TITLE
Datapusher Error

### DISCUSSION THREAD

**Discussion by mnnnmnnnm** [ckan/ckan#7023] [Q&A]
CKAN : 2.9.5
Built using Docker-Compose.

Hello, I had an error in Datastore while I was doing CKAN. I don't know how to solve it. Help

<img width="951" alt="image" src="https://user-images.githubusercontent.com/111728309/185852749-aa5a7ef3-028b-4294-beff-a858e3587473.png">

<img width="872" alt="image" src="https://user-images.githubusercontent.com/111728309/185852852-17f9100e-4a06-4d40-bf9d-60dc37e69eee.png">

**Comment by kowh-ai** [comment]
`'Connection refused'` generally means the **host** _and/or_ the **port** you are trying to connect to is down _or_ misconfigured
